classes/forms/plagiarism_copyleaks_adminform.class.php

```diff
@@ -255,6 +255,19 @@ public function save(stdClass $data) {
 
         $cache = cache::make('core', 'config');
         $cache->delete('plagiarism_copyleaks');
-        \plagiarism_copyleaks_comms::test_copyleaks_connection('admin_settings_page', true);
+        if (\plagiarism_copyleaks_comms::test_copyleaks_connection('admin_settings_page', true)) {
```

Comment:
Won't this affect new integrations with on setup, where Key and Secret still need to be set?